aws-py-voting-app/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pulumi>=2.0.0,<3.0.0
+pulumi-aws>=2.0.0,<3.0.0
+pulumi-docker>=1.0.0,<3.0.0
```

Comment:
we should make this `pulumi-docker>=2.0.0,<3.0.0`

Reply:
Done!